go/servenv/logging.go

```diff
@@ -228,18 +228,6 @@ func fireLoggingSetupHooks(l *slog.Logger) {
 	}
 }
 
-// fireLoggingChangeHooks calls all registered logging change hooks.
```

Comment:
This one, we should check what did we miss to port from vitess. I'm curious where this was being called. 

It might be useful and we are missing porting parts of servenv.

Reply:
This function seems to only exist in multigres: https://github.com/search?q=fireLoggingChangeHooks&type=code